components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -227,6 +229,13 @@ function CustomLeague:defineCustomPageVariables(args)
 	-- Module:Prize pool, Module:Prize pool team, Module:TeamCard and Module:TeamCard2
 	Variables.varDefine('tournament_deadline', DateClean(args.deadline or ''))
 	Variables.varDefine('tournament_gamemode', table.concat(CustomLeague:_getGameModes(args, false), ','))
+
+	-- map links, to be used by brackets and mappool templates
+	local _
+	_, _maps = CustomLeague:_getMaps(args)
```

Comment:
Is there a way to do this a bit cleaner?
I could think about moving the creation of the first returned table to the parse function.

Reply:
Right, but that isn't what i meant.
I didn't like to write
```
local _
_, variable = call()
```
So to define an unused var on an extra line

Reply:
By convention, `_`doesn't need to be declared local, as it's just junk.

Reply:
```suggestion
	_, _maps = CustomLeague:_getMaps(args)
```

Reply:
> By convention, _doesn't need to be declared local, as it's just junk.
Codestyle check doesn't like it: https://github.com/Liquipedia/Lua-Modules/runs/7967905777